pages/common/axel.md

```diff
@@ -0,0 +1,24 @@
+# axel
+
+> Download accelerator
+> Supports HTTP, HTTPS, and FTP
+
+- Download a URL to a file
+
+`axel {{url}}`
+
+- Download and specify filename
+
+`axel {{url}} -o {{filename}}`
+
+- Download with multiple connections
+
+`axel -n {{connections_num}} {{url}}`
+
+- Search for mirrors (can specify numbers)
```

Comment:
I think "(can specify numbers)" can be omitted, since there's `{{mirrors_num}}` in the example already.